.travis.yml

```diff
@@ -48,9 +47,9 @@ before_script:
 
 script:
   - set -ev
-  - mvn install -Drelease
+  - mvn clean install -Drelease
```

Comment:
It can be argued that it's good practise but if you feel it's wholly unnecessary it can be removed.